source/apiVolontaria/apiVolontaria/factories.py

```diff
@@ -10,8 +10,8 @@ class Meta:
 
     first_name = factory.Faker('first_name')
     last_name = factory.Faker('last_name')
-    username = factory.Sequence('John{0}'.format)
-    email = factory.Sequence('john{0}@volontaria.com'.format)
+    username = factory.Faker('email')
+    email = factory.Faker('email')
```

Comment:
`username` and `email` are unique, we will have some errors if `Faker` return an existing value.

Reply:
Yes done

Reply:
Hmmm I see... Is the previous solution was working ?